controllers/che/checluster_controller.go

```diff
@@ -103,6 +102,19 @@ func NewReconciler(
 	reconcileManager.RegisterReconciler(rbac.NewCheServerPermissionsReconciler())
 	reconcileManager.RegisterReconciler(rbac.NewGatewayPermissionsReconciler())
 	reconcileManager.RegisterReconciler(rbac.NewWorkspacePermissionsReconciler())
+	reconcileManager.RegisterReconciler(server.NewDefaultValuesReconciler())
+
+	// we have to expose che endpoint independently of syncing other server
+	// resources since che host is used for dashboard deployment and che config map
+	reconcileManager.RegisterReconciler(server.NewCheHostReconciler())
+	reconcileManager.RegisterReconciler(postgres.NewPostgresReconciler())
+	reconcileManager.RegisterReconciler(identityprovider.NewIdentityProviderReconciler())
+	reconcileManager.RegisterReconciler(devfileregistry.NewDevfileRegistryReconciler())
+	reconcileManager.RegisterReconciler(pluginregistry.NewPluginRegistryReconciler())
+	reconcileManager.RegisterReconciler(dashboard.NewDashboardReconciler())
+	reconcileManager.RegisterReconciler(gateway.NewGatewayReconciler())
+	reconcileManager.RegisterReconciler(server.NewCheServerReconciler())
+	reconcileManager.RegisterReconciler(consolelink.NewConsoleLinkReconciler())
```

Comment:
Minor: Maybe, we can check here if Openshift 4, then register reconsiler.

Reply:
Make sense